packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigation.tsx

```diff
@@ -137,7 +141,7 @@ export const CarouselNavigation: ComponentWithAs<'ul', CarouselNavigationProps>
             vertical,
             thumbnails,
           }),
-        overrideProps: handleItemOverrides(variables),
+        overrideProps: disableClickableNav ? undefined : handleItemOverrides(variables),
```

Comment:
I would probably pass it down to the `CarouselNavigationItem ` as a prop and there on `handleClick` return if the flag is enabled.

Reply:
yes, we should always call handleItemOverrides, otherwise it gets unpredictable

Reply:
Done, thanks